DataFormats/Scouting/interface/Run3ScoutingElectron.h

```diff
@@ -13,11 +13,18 @@ class Run3ScoutingElectron {
                        float eta,
                        float phi,
                        float m,
+                       float rawEnergy,
```

Comment:
just leaving a comment for possible further discussion here (apologies if it was already brought up elsewhere). I am wondering if with the growing list of arguments this style of constructor might become error-prone.

Reply:
I tend to agree this pattern at this number of arguments can become error prone. The mixture of types (especially of fundamental and `vector` types) at least forces all callers of the constructor to be updated, making a mistake on the order of same type (or implicitly convertible) arguments is nevertheless quite easy.

Just to think out loud, in C++20 aggregate classes ("structs") can be constructed with designated initializers, which would allow python keyword argument-style construction in the EDProducers. That would require making all the member data public, which would be a bit atypical. On the other hand, the scouting classes look like collection of various data without any class invariants, so in principle having the member data public could be ok.

Reply:
@arsahasransu why was this resolved?